__tests__/app.test.js

```diff
@@ -27,7 +27,7 @@ describe("/api/topics", () => {
       .expect(200)
       .then(({ body }) => {
         body.topics.forEach((topic) => {
-          expect(topic).toMatchObject({
+          expect(topic).toEqual({
```

Comment:
Your jest matcher was fine!
Think about how the forEach works - if there are no elements in the array, the callback fn in the forEach will not be invoked - therefore no expect statements will be called and the test will pass by default. Your test needs to make sure that there is actually something in the array first! (Hint: .length)